docs(how-to): remote access feature - MTA-1858 (#514)
scaleway/docs-content: compute/elastic-metal/how-to/activate-remote-access.mdx

---
meta:
  title: How to activate remote access on an Elastic Metal server
  description: This page explains how to activate remote access on an Elastic Metal server
content:
  h1: How to activate remote access on an Elastic Metal server
  paragraph: This page explains how to activate remote access on an Elastic Metal server
tags: advanced settings
dates:
  validation: 2022-03-30
  posted: 2022-03-30
---

> Remote access is available on most Elastic Metal servers as option. This option allows you to access the keyboard, video, and mouse of the machine remotely thorough a KVM-over-IP device. 

Reviewer: ```suggestion
Remote access is available on most Elastic Metal servers as an option. This option allows you to access the keyboard, video, and mouse of the machine remotely through a KVM-over-IP device. 
```

> Using remote access you can access to the machine for debugging purposes, even if the installed OS is not working properly. It is also possible to install an [operationg system](/compute/elastic-metal/how-to/install-server-from-kvm) on the machine using a remote ISO file.

Reviewer 2: ```suggestion
Using remote access you can access to the machine for debugging purposes, even if the installed OS is not working properly. It is also possible to install an [operating system](/compute/elastic-metal/how-to/install-server-from-kvm) on the machine using a remote ISO file.
```

<Message type="requirement">

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
- You have [created](/compute/elastic-metal/how-to/create-server) an Elastic Metal server

</Message>

## Activating remote acccess

1. Click **Elastic Metal** in the **Compute** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Elastic Metal dashboard displays.

2. Click the name of the server you want to access using the remote access option.

   <Lightbox src="assets/scaleway-elastic-metal-list.webp" alt="" />

> 3. On the server's information page, scroll down to the **Remote Access Feature**. Then click **Activate**.

Reviewer 2: ```suggestion
3. Scroll down to the **Remote Access Feature** in the server's information page. Then click **Activate**.
```

  <Lightbox src="scaleway-remote-access-feature.webp" alt="" />

4. Read and accept the terms and conditions for the remote access feature. Tick the box to confirm you understood the conditions of the option, then click **Activate**.

  <Lightbox src="scaleway-activate-remote-access.webp" alt="" />

5. Enter the [IP address](https://www.whatsmyip.org/) of your Internet connection and click **Create access**.

  <Lightbox src="scaleway-remote-access-ip.webp" alt="" />

  <Message type="note">

>   Make sure to enter the public IP address of your Internet connection, as connections to the remote access device are limited to this IP address for security reasons. Currently only IPv4 addresses are supported by the remote access feature. 

Reviewer: ```suggestion
  Make sure to enter the public IP address of your Internet connection, as connections to the remote access device are limited to this IP address for security reasons. Currently, only IPv4 addresses are supported by the remote access feature. 
```
